rasa/nlu/utils/hugging_face/hf_transformers.py

```diff
@@ -0,0 +1,328 @@
+import logging
+from typing import Any, Dict, List, Text, Tuple, Optional
+
+from rasa.nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+from rasa.nlu.components import Component
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.training_data import Message, TrainingData
+from rasa.nlu.tokenizers.tokenizer import Token
+from rasa.utils.train_utils import align_tokens
```

Comment:
Can we just do `import rasa.utils.train_utils` and then do `train_utils.align_tokens` in the code. Same for ConveRT.

Reply:
Curious, any best practice involved there?

Reply:
Yes. Listed on our Notion page: https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e

Reply:
Got it! Thanks